app/settings/template.hbs

```diff
@@ -77,12 +77,12 @@
                             </nav.link-to>
                         </nav.item>
                         <nav.item>
-                            <OsfLink
+                            <nav.link-to
```

Comment:
Drive-by: This should be an [OsfLink](https://centerforopenscience.github.io/ember-osf-web/handbook/docs/components/osf-link). It can transition to routes as well as using hrefs.

Reply:
Is it smart enough to know when to route via the ember app? When I had it there before it was reloading the entire page (not ideal). Is there a different parameter than href to not reload the app?

Reply:
Yeah, check out the docs I linked to. You use different parameters for link versus link-to behavior.

Reply:
And I suspect we're going to have to add in support for feature-flagged routes before any of these pages ship, and that's easier to do in the OsfLink generally than at each item.

Reply:
Cool, did not see the link ^^

Reply:
No, you're totally right. They need to be nav/link-to's so ember-bootstrap knows what to do. Sorry for the diversion.

Reply:
No worries, I'll revert for now but surely we'll be able to just highlight sections with the OsfLink (probably not a huge task)

Reply:
There's a lot of bookkeeping that ember-bootstrap does to check the dom hierarchy and then update a variable. We might be able to hack it, but there didn't look like a good way when we looked at fixing it. But if you can find a way, that would be awesome to not need those.

Reply:
Drive-by: For styling based on the currently active route (I think that's the highlight you're talking about), [OsfLink puts `class='active'` on the anchor tag](https://github.com/CenterForOpenScience/ember-osf-web/blob/a9756531e95ddffdede1e37c7717408da9eae426/lib/osf-components/addon/components/osf-link/template.hbs#L5). It's not a drop-in replacement because bootstrap expects `active` on the `<li>`, but I'm sure we could style it ourselves if we choose to.

Reply:
Good point. This works for single-level items, but would not work for the settings-profile hierarchy, which has an additional level.

Reply:
Maybe we can revisit the settings panel to accommodate the OsfLink in another PR - would be nice to have this functionality in.